src/js/components/Video/Video.js

```diff
@@ -282,7 +289,7 @@ const Video = forwardRef(
               <Box flex>
                 <Stack>
                   <Meter
-                    aria-label="Video progress"
+                    aria-label={messages.progressMeter || 'Video progress'}
```

Comment:
Since we are providing a default value to `messages.progressMeter`, I don't think there is a point using the OR operator here.

Reply:
good catch!